baseplate/context/cqlmapper.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from cqlmapper.connection import Connection as CQLConnection
```

Comment:
I always get kinda confused by these import-and-rename things. How about just doing `import cqlmapper.connection` and using `return cqlmapper.connection.Connection` if you want it scoped or just go straight up unmangled `Connection`.